selfdrive/car/hyundai/values.py

```diff
@@ -11,11 +11,11 @@ def get_hud_alerts(visual_alert, audible_alert):
     return 0
 
 class CAR:
-  SANTA_FE = "HYUNDAI SANTA FE LIMITED 2019"
-  ELANTRA = "HYUNDAI ELANTRA LIMITED ULTIMATE 2017"
-  GENESIS = "HYUNDAI GENESIS 2018"
-  KIA_SORENTO = "KIA SORENTO GT LINE 2018"  # Top Trim Kia Sorento for Australian Market, AWD Diesel 8sp Auto
-  KIA_STINGER = "KIA STINGER GT2 2018"
+  ELANTRA = "HYUNDAI ELANTRA LIMITED ULTIMATE 2017"   # First User @TK211X    <-- Ported by @ku7 (Second, First Hyundai in World)
```

Comment:
What's the reason for all these changes?

Reply:
done